perf_test/ode/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+KOKKOSKERNELS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+KOKKOSKERNELS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+
+if(KOKKOSKERNELS_ENABLE_BENCHMARK)
+  SET(BENCHMARK_NAME ${PACKAGE_NAME}_ode_runge_kutta)
+
+  ADD_EXECUTABLE(
+      ${BENCHMARK_NAME}
+      KokkosODE_RK.cpp
+  )
+  TARGET_LINK_LIBRARIES(
+      ${BENCHMARK_NAME}
+      PRIVATE benchmark::benchmark Kokkos::kokkoskernels
+  )
+  TARGET_INCLUDE_DIRECTORIES(
+      ${BENCHMARK_NAME}
+      SYSTEM PRIVATE ${benchmark_SOURCE_DIR}/include
+  )
+  ADD_TEST(
+      NAME ${BENCHMARK_NAME}
+      COMMAND ${BENCHMARK_NAME}
+  )
+else()
+  KOKKOSKERNELS_ADD_EXECUTABLE(
```

Comment:
If KokkosODE_RK.cpp requires benchmark, this ADD_EXECUTABLE should be gone right?

Reply:
I enabled benchmark and tried playing around with the perf test, but it seems like some command line arguments aren't having the expected effect?
```
# Both of these take about 200 ms to run
./KokkosKernels_ode_runge_kutta --repeat 100 --n 10000
./KokkosKernels_ode_runge_kutta --repeat 1 --n 10000
```


```
./KokkosKernels_ode_runge_kutta --cuda 00
Unrecognized command line argument #1: --cuda
```
This one might not even be intended to work, in which case perf_test::list_common_options() probably shouldn't be in print_options()

and I think model 2 is assuming that the memory space is host accessible:
```
./KokkosKernels_ode_runge_kutta --model 2
Kokkos::View ERROR: attempt to access inaccessible memory space (label="initial conditions")
```